memcached.yaml

```diff
@@ -139,18 +150,23 @@ test:
             openssl req -new -key example.key -out example.csr -subj "/CN=example.com" -config openssl.cnf
             openssl x509 -req -days 2 -in example.csr -signkey example.key -out example.crt
 
-            memcached -Z -o ssl_chain_cert=example.crt,ssl_key=example.key&
+            # Find free ports for memcached and stunnel.
+            # Avoids intermittent port conflicts errors that were occurring in CI runs.
+            MEMCACHED_PORT=$(for p in $(shuf -i 30000-40000 -n 10); do ss -ltn | awk '{print $4}' | grep -q ":$p\$" || { echo $p; break; }; done)
+            ACCEPT_PORT=$(for p in $(shuf -i 30000-40000 -n 10); do ss -ltn | awk '{print $4}' | grep -q ":$p\$" || { echo $p; break; }; done)
```

Comment:
These aren't necessarily going to be free ports every single time, but definitely more reliable